<p>This rule detects direct usage of Html in Controller. In Hybris Controller should not define the HTML directly. </p>
<h2>Noncompliant Code Example</h2>
<pre>
@Controller
public clall HtmlUsageTestSampleController {
	String nonCompliantMethod() {
        return "<html><body></body></html>"; // Noncompliant {{Refactor Controller to not use html tags directly and let view template define the tags}}
    }
}
</pre>
<h2>Compliant Solution</h2>
<pre>
@Controller
public class HtmlUsageTestSampleController {
...
    String compliantMethod() {
        return "index.jsp";
    }
...

}
</pre>